src/figuresClasses.ts

```diff
@@ -1,19 +1,88 @@
-export interface Figure {
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shapes;
+  color: Color;
+  getArea: () => number;
 }
 
-export class Triangle {
+function sidesLengthCheck(...sides: number[]): void {
```

Comment:
IMO, function names are looking better when you place `action` before the `subject`, like actionSubject, createArray, etc.
```suggestion
function checkSidesLength(...sides: number[]): void {
```